File: DataInjectorIOS/Example/DataInjector/helper/Bitlets.swift
//
//  Bitlets.swift
//  DataInjector example
//
//  Example helper: the list of bitlets used in the app
//

import Foundation

class Bitlets {

    static let customers = MockBitlet(filename: "customer_list", cacheKey: "customers")
    static let products = MockBitlet(filename: "product_list", cacheKey: "products")

}

File: DataInjectorAndroid/DataInjectorExample/src/main/java/com/crescentflare/datainjectorexample/helper/Bitlets.java
package com.crescentflare.datainjectorexample.helper;

import com.crescentflare.datainjectorexample.R;

/**
 * The list of bitlets used in the app
 */
public class Bitlets
{
    public static MockBitlet customers = new MockBitlet(R.raw.customer_list, "customers");
    public static MockBitlet products = new MockBitlet(R.raw.product_list, "products");
}
